lib/Dialect/Torch/Transforms/Passes.cpp

```diff
@@ -86,6 +91,15 @@ void mlir::torch::Torch::createTorchFunctionToTorchBackendPipeline(
       options.backendLegalOps, options.extraLibrary));
 }
 
+void mlir::torch::Torch::createTorchOnnxToTorchBackendPipeline(
+    OpPassManager &pm, const TorchLoweringPipelineOptions &options) {
+  pm.addPass(onnx_c::createTorchOnnxToTorchPass());
+  // The above pass just converts the torch onnx IR to torch, hence the given
+  // pipeline will make sure that the IR is transformed such that it satisfies
+  // the backend contract.
+  createTorchFunctionToTorchBackendPipeline(pm, options);
```

Comment:
We don't need most of the passes in this pipeline. I'd like to start with an initial `--torch-decompose-complex-ops`, then let's make a separate `--onnx-shape-refinement-pipeline` to put here that iteratively performs the following: 

`--torch-scalarize-shapes` and `--torch-shape-refinement-pipeline`. 

Then we will follow up with:

`--torch-refine-public-return`
`--canonicalize`
`--torch-decompose-complex-ops`
`--canonicalize`